doc/files.md

```diff
@@ -644,6 +645,21 @@ file.createMetadata(new Metadata().add("/foo", "bar"));
 [create-metadata-2]: http://opensource.box.com/box-java-sdk/javadoc/com/box/sdk/BoxFile.html#createMetadata-java.lang.String-com.box.sdk.Metadata-
 [create-metadata-3]: http://opensource.box.com/box-java-sdk/javadoc/com/box/sdk/BoxFile.html#createMetadata-java.lang.String-java.lang.String-com.box.sdk.Metadata-
 
+Set Metadata
```

Comment:
Similar to how we've documented other set methods that wrap POST/PUT calls, I think we should move the existing "Create Metadata" and "Update Metadata" docs under this section as additional information.  I expect most people will want to use the new set method anyways, and we should make sure that they get that information up front.